### PR TITLE
PaleoIngredientCheck: Converting to text template

### PR DESCRIPTION
**Pull request by mintsoft**
###### Description of changes

Ref: https://github.com/duckduckgo/zeroclickinfo-goodies/issues/2767

---

Instant Answer Page: https://duck.co/ia/view/paleo_ingredient_check

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @javathunderman

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: lib/DDG/Goodie/PaleoIngredientCheck.pm [hunk 1]
```diff
@@ -38,11 +38,14 @@ handle remainder => sub {
...
 
-    return $result,
-    structured_answer => {
-        input     => [$item], # or just the original query
-        operation => "Paleo Friendly",
-        result    => $result
+    return $result, structured_answer => {
+        data => {
+            title => $result,
+            subtitle => "Paleo Friendly: $item"
```
Comment (GuiltyDolphin) [inline lib/DDG/Goodie/PaleoIngredientCheck.pm]: Might we have to `enc` this? Looks like anything can go through here?

Comment (mintsoft) [inline lib/DDG/Goodie/PaleoIngredientCheck.pm]: I'll enc it for protection; it's not "needed" but it is needed I think. I'm sure @moollaza can clarify when he's back

**Comment by GuiltyDolphin** [comment]
@mintsoft Thanks!

File: lib/DDG/Goodie/PaleoIngredientCheck.pm (hunk 1 above)
Comment (moollaza) [inline lib/DDG/Goodie/PaleoIngredientCheck.pm]: Is there any chance of `item` containing non alpha-numeric character? I don't think so, in which case we don't need the HTML enc

Comment (GuiltyDolphin) [inline lib/DDG/Goodie/PaleoIngredientCheck.pm]: @moollaza It looks like pretty much anything can get through; non alpha-nums will just be non-friendly.

Comment (GuiltyDolphin) [inline lib/DDG/Goodie/PaleoIngredientCheck.pm]: @moollaza @mintsoft No wait... I think I'm wrong; looks like it's white-listing certain phrases.

**Pull request merged**